guidelines.md

```diff
@@ -60,25 +60,41 @@ Some errors may also contain additional information relevant to the error on top
 
 Authentication and tokens that have to be used depending on environment.
 
-### Demo environment
+### Demo environments
 
-This environment is meant to be used during implementation of the client applications.
+These demo environments are meant to be used during implementation of the client applications. To sign into the system, use the following credentials:
 
 * **PlatformAddress** - `https://demo.mews.li`
+* **Email** - `connector-api@mews.li`
+* **Password** - `connector-api`
+
+The above credentials will allow access to both pricing environments.
```

Comment:
Probably, we should mention, that you are able to switch the property in the right corner, otherwise they will have no idea, in which property they currently are. (They will not see payments, reservations,...)